src/core/Microsoft.PowerToys.Settings.UI/Views/ImageResizerPage.xaml

```diff
@@ -143,7 +168,6 @@
                                             x:Uid="ImageResizer_Height_Property"
                                             Value="{x:Bind Path=Height, Mode=TwoWay}" 
                                             Width="68"
-                                            Height="34"
```

Comment:
I'd go for MinHeight instead of removing it entirely to make sure it renders properly when setting display scaling to e.g. 100%/150%